Project-2022-23/Tauben
# Task/i18n

# Pull-/Merge-Request
#### Task-Nr. RP-20 i18n

## Description of the Changes in this Branch
The translation of the WebSite is implemented. Translations need to be written by @nick-usw 

## Test-Cases
1. Test-case
    - change the Language
    - See if it is translated (only on home available so far, only translation)
